library/std/src/sys/vxworks/ext/fs.rs

```diff
@@ -75,8 +75,8 @@ pub trait FileExt {
     /// has read, but it will never read more than would be necessary to
     /// completely fill the buffer.
     ///
-    /// [`ErrorKind::Interrupted`]: ../../../../std/io/enum.ErrorKind.html#variant.Interrupted
-    /// [`ErrorKind::UnexpectedEof`]: ../../../../std/io/enum.ErrorKind.html#variant.UnexpectedEof
+    /// [`ErrorKind::Interrupted`]: crate::io::ErrorKind::Interrupted
+    /// [`ErrorKind::UnexpectedEof`]: crate::io::ErrorKind::UnexpectedEof
```

Comment:
`io` is in scope here: https://github.com/rust-lang/rust/blob/9463380b0ceeca6c99d7466f90f1bf9fe4732d8d/library/std/src/sys/vxworks/ext/fs.rs#L4

```suggestion
    /// [`ErrorKind::Interrupted`]: io::ErrorKind::Interrupted
    /// [`ErrorKind::UnexpectedEof`]: io::ErrorKind::UnexpectedEof
```